### PR TITLE
Add warning about OCP version support

### PR DESCRIPTION
**Pull request by leifmadsen**

### CODE REVIEW

**Comment by leifmadsen** [comment]
[Import and release upstream 1.0.3 into downstream for OCP 4.4 support](https://trello.com/c/G4hqZbdz/1043-import-and-release-upstream-103-into-downstream-for-ocp-44-support)

**Comment by pkilambi** [comment]
👍

File: doc-Service-Telemetry-Framework/assemblies/assembly_installing-the-core-components-of-stf.adoc [hunk 1]
```diff
@@ -43,6 +43,11 @@
 
 Before you install {Project} ({ProjectShort}), ensure that {OpenShift} ({OpenShiftShort}) version 4.x is running and that you understand the core components of the framework. As part of the {OpenShiftShort} installation planning process, ensure that the administrator provides persistent storage and enough resources to run the {ProjectShort} components on top of the {OpenShiftShort} environment.
 
+ifeval::["{build}" == "downstream"]
+[WARNING]
+{OpenShift} version 4.3 is currently required for a successful installation of {ProjectShort}. The release of Smart Gateway Operator 1.0.2 will result in support of {OpenShift} version 4.4 and later.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/assemblies/assembly_installing-the-core-components-of-stf.adoc:48]: Hi Leif, is this a temporary warning? Until 16.1 z1?

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/assemblies/assembly_installing-the-core-components-of-stf.adoc:48]: Yes, temporary. I would say 16.1 z1 is probably the appropriate target. I'm hoping for it to be a bit sooner than that if possible. The downstream build is complete, I'm just waiting for it to work through the release process.

**Pull request merged**